docs/source/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow: /
+Allow: /en/stable/
```

Comment:
Github seems to be complaining about no newline at the end of this file.
```suggestion
Allow: /en/stable/

```

Reply:
Hmm, I can't commit this because it's "identical to original text".